packages/cursorless-vscode/src/keyboard/grammar/getAcceptableTokenTypes.ts

```diff
@@ -1,9 +1,28 @@
 import nearley, { State } from "nearley";
-import { isEqual } from "lodash";
+import { isEqual, times } from "lodash";
 import { CommandRulePostProcessor } from "./CommandRulePostProcessor";
-import { UniqueWorkQueue } from "./UniqueWorkQueue";
-import { uniqWithHash } from "@cursorless/common";
+import { DefaultMap, uniqWithHash } from "@cursorless/common";
 import { KeyboardCommandHandler } from "../KeyboardCommandHandler";
+import { TokenType } from "../TokenTypeHelpers";
+
+export interface AcceptableTokenType {
+  /**
+   * The token type, eg "color".
+   */
+  type: TokenType;
+
+  /**
+   * The command that wants the token type
+   */
+  command: keyof KeyboardCommandHandler;
+
+  /**
+   * A partial argument for the command that wants the token type, where
+   * {@link NEXT} indicates where the next token would go and {@link MISSING}
```

Comment:
Yeah. It's special because we could potentially use it in an interesting way in the sidebar. When we construct all partial args in response to a hypothetical next token type, we could then enumerate the possible tokens for that type and slot them into `NEXT` to see what they'd look like. That way we could have a sidebar section for eg `color`, and then show what the command would be for each different color you could type